.craft.yml

```diff
@@ -9,10 +9,12 @@ targets:
       flutter:
       logging:
       dio:
+      file:
   - name: github
   - name: registry
     sdks:
       pub:sentry:
       pub:sentry_flutter:
       pub:sentry_logging:
       pub:sentry_dio:
+      #pub:sentry_dio:
```

Comment:
Double check if craft is already able to auto create the first entry in release registry

Reply:
@adinauer @kamilogorek what's the outcome of that problem, do we still need to do the release registry dance manually or is it fine now?
https://github.com/getsentry/sentry-release-registry/pull/85

Reply:
Yes, it's still "broken"